src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,33 @@
+package duke;
+
+import duke.command.*;
+
+public class Parser {
+
+    /**
+     * Parses user input and returns the appropriate Command
+     *
+     * @param fullCommand user input
+     * @return appropriate Command
+     */
+    public static Command parse(String fullCommand) {
+        if (fullCommand.equals("bye")) {
```

Comment:
Maybe you can consider using a switch statement here to make the code look cleaner?